Updates to video transcripts (#3753)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

It's important that you give yourself time to write the best possible [personal statement](/how-to-apply-for-teacher-training/teacher-training-personal-statement). Providers read your personal statement carefully as part of their assessment of your application.

> Your personal statement should be between 500 and 1000 words long. 90% of successful candidates write 500 words or more.

Reviewer: ```suggestion
Your personal statement can be up to 1000 words. 90% of successful candidates write 500 words or more.
```
This change seems to be have been brought in erroneously so putting back to what is in production.
